src/main.js

```diff
@@ -79,4 +83,5 @@ function displayFormSubmissionCover(event) {
   currentCover = new Cover(inputCoverImage.value, inputTitle.value, inputDesc1.value, inputDesc2.value);
   formViewPage.classList.add('hidden');
   mainCoverPage.classList.remove('hidden');
+  saveCoverButton.classList.remove('hidden');
```

Comment:
This was also something we didn't think about doing, but when Nicole saw there was only three buttons we knew we had to display this button.